wiki/Tournaments/ASC/2/en.md

```diff
@@ -0,0 +1,208 @@
+
+## Ruleset
+
+### General rules
+
+1. This is a **3v3 double-elimination tournament**, with a team size of 6 and a total of 16 teams. Only the top 96 players will be chosen to join this edition.
+2. The tournament is open to all players who have **country flags from Asia and Oceania** in their profiles. Participants must have flags from these regions to be eligible for entry.
+3. To sign up players must do so through the website and must be in the discord server. Failure to do so will count your registration as incomplete and not eligible to participate.
+4. Players display names in the Discord server must be the same as their osu! username during the entire duration of the tournament.
+5. A "sandbagging" formula will be used to help determine the true ranks of players to avoid disruptions and highly unbalanced teams that may potentially ruin the Suiji Cup experience. Player seedings and formula details will be revealed right before sign ups close.
+6. Participants will be split up into 3 different seeds, "A", "B" and "C". Each team will be formed by a live randomization process and will consist of 2 players from each seed (a team will have 6 players, A, A, B, B, C, C).
+7. "A" seeded players have the ability to substitute with "B" or "C" seed on their team. "B" seeded players also have the ability to substitute but only with "C" seed on their team.
+8. Teams will be given 5 days to choose a team name and a captain. Failure to submit a team name and a captain will result in an assigned team name chosen by the staff and the highest-ranked player being made captain by default.
+
+### Additional information
+
+1. Unlike the previous iteration, there will only be 1 SV pick in the mappool, and it cannot be banned.
+2. Visual mods such as: HD, FL, FI, and MR along with NF mod are allowed. Meanwhile, EZ and HR mod are not allowed.
+3. In the event of a disconnection, the map can be replayed if the disconnect is within the first 30 seconds or before the first quarter of the map. Otherwise, if there is proof of the disconnected player's score, it will be used. Each team is allowed one replay per match, the referee will have the final say in disconnection situations.
+4. If a player is not cooperative with the referee and intentionally stalls the match, they will be warned by the referee. If the problem persists the host has the power to end the match and decide the winner on the spot.
+5. Other special and unique cases that could happen will be up to the host to decide the next moves.
+
+### Schedules
+
+We, the Tournament staff will be responsible for scheduling all matches and informing players when they are scheduled to play. If a player is unable to play with the schedule they will have the option to request a reschedule, details about this topic will be explained further when the tournament begins.
```

Comment:
It is preferable not to speak in first person in the wiki. It must remain as neutral as possible.
```suggestion
The tournament staff will be responsible for scheduling all matches and informing players when they are scheduled to play. If a player is unable to play with the schedule they will have the option to request a reschedule, details about this topic will be explained further when the tournament begins.
```